extensions/version-fetcher/get-latest-operator-version.js

```diff
@@ -0,0 +1,32 @@
+// Fetch the latest release version from GitHub
+const { Octokit } = require("@octokit/rest");
+const { retry } = require("@octokit/plugin-retry");
+const OctokitWithRetries = Octokit.plugin(retry);
+const owner = 'redpanda-data';
+const repo = 'redpanda-operator';
+
+let githubOptions = {
+  userAgent: 'Redpanda Docs',
+  baseUrl: 'https://api.github.com',
+};
+
+if (process.env.REDPANDA_GITHUB_TOKEN) {
+  githubOptions.auth = process.env.REDPANDA_GITHUB_TOKEN;
+}
+
+const github = new OctokitWithRetries(githubOptions);
+
+var latestOperatorReleaseVersion;
+
+module.exports = async () => {
+  await github.rest.repos.getLatestRelease({
```

Comment:
It's a bit mind bending to me to see `await` intermixed with `.then`s. Is there a reason for that?
It also seems like `latestOperatorReleaseVersion` is meant to be a cache but it's not utilized as such?

Do you think something like this would work?
```js
// Used cached value, if set.
if (!!latestOperatorReleaseVersion) return latestOperatorReleaseVersion;

try {
    let release = await GitHub.rest.repos.getLatestRelease({owner, repo});
    latestOperatorReleaseVersion = release.data.tag_name;
    return latestOperatorReleaseVersion;
} catch (error) {
    console.error(error)
    return null;
}
```

Reply:
Thanks, Chris - updated to use `await` without the `then()`. Those values weren't cached. It runs once per build of the docs.